traffic_ops/testing/api/v2/tc-fixtures.json

```diff
@@ -717,6 +717,12 @@
             "typeName": "HOST_REGEXP",
             "setNumber": 1,
             "pattern" : ".*"
+        },
+        {
+            "dsName": "ds1",
+            "typeName": "HOST_REGEXP",
+            "setNumber": 3,
+            "pattern" : ""
         }
```

Comment:
Why we do we allow empty regexes? It should have a nonzero length, right?

Reply:
I see, by putting it in `tc-fixtures.json`, you get all of the IDs for different tables it is related to. In the future, it might be nice to have a separate JSON file for invalid fixtures for negative tests so we can assume that everything in `tc-fixtures.json` is valid, but we don't have that yet, so the way you did it makes sense.